hack/ci-kind-molecule-tests.sh

```diff
@@ -90,7 +90,7 @@ Options:
 
 -lb|--logs-branch <branch name>
     The logs branch to clone.
-    Default: master
+    Default: kind
```

Comment:
Should this still be `kind`?

Reply:
yes, this is the cluster kind branch for the log repo.. See: https://github.com/jmazzitelli/kiali-molecule-test-logs/branches

You can change the branch name if you want to put logs in a different repo and you want a different branch.

Reply:
Because this is running molecule tests on KinD, I put the logs in the "kind" branch. You will see I also have minikube and openshift branches for when I run nightly tests on those clusters